toolkit/package.json

```diff
@@ -25,6 +25,8 @@
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",
     "lint:types": "glint",
+    "toolkit:publish": "yalc publish --content",
+    "toolkit:cleanup": "yalc installations clean @hashicorp/consul-ui-toolkit",
```

Comment:
Some thoughts about changing the naming of these. None of the other yarn scripts in the `toolkit/package.json` have the prefix `toolkit` as they are already in the `toolkit`. It also might be good to have the script name be clear that it's not actually publishing to npm.  Maybe something like `publish:local` or `publish:yalc` or something along those lines.

Reply:
I don't mind the `toolkit:` prefix for my second suggestion, which is to add scripts to the root `package.json` to run these from the root. 

Example:
```
// in the root package.json scripts
"toolkit:publish:local": "yarn workspace @hashicorp/consul-ui-toolkit run toolkit:publish"
```

That way you can publish and cleanup from the root. Keeping the prefix here would make more sense as it lets you know where the command is being run.. however the other ones either use a post fix (`start:documentation`) or don't have either (`build`).

Reply:
agree 👍 I thought a lot of the naming here 😅 

thank you for the ideas 💡